.github/workflows/integration-test-src.yml

```diff
@@ -65,7 +65,7 @@ jobs:
 
   java-sdk-cluster-memory-0:
     if: ${{ github.event.inputs.EXEC_TEST_TYPE == 'all' || github.event.inputs.EXEC_TEST_TYPE == 'java' }}
-    runs-on: ubuntu-latest
+    runs-on: self-hosted
```

Comment:
`runs-on: [self-hosted,generic]` if use docker
`runs-on: [self-hosted,common-user]` for others